src/modules/whitelist/main_impl.h

```diff
@@ -47,19 +47,7 @@ int secp256k1_whitelist_sign(const secp256k1_context* ctx, secp256k1_whitelist_s
         secp256k1_scalar_get_b32(seckey32, &sec);
         while (1) {
             size_t i;
-            unsigned char nonce32[32];
-            int done;
-            ret = secp256k1_nonce_function_default(nonce32, msg32, seckey32, NULL, NULL, count);
-            if (!ret) {
-                break;
-            }
-            secp256k1_scalar_set_b32(&non, nonce32, &overflow);
-            memset(nonce32, 0, 32);
-            if (overflow || secp256k1_scalar_is_zero(&non)) {
-                count++;
-                continue;
-            }
-            done = 1;
```

Comment:
nit: I believe this change is fine. But in general, I'm not worth making similar changes if the only short term goal is to reduce stack usage and and if we're going to replace this anyway.

Reply:
This removed logic is incompatible with moving the nonce-computation logic into `borromean_sign`, which is (a) necessary to do any useful amount of stack space reduction; (b) reduces the reviewing burden by no longer requiring that array indices of secret data in far-away files be compatible.

Reply:
I'm also unsure what you mean by "we're going to replace this anyway"